tests/tools/get_tests_inorder.sh

```diff
@@ -18,4 +18,4 @@ generated=$(for j in `ls */*.testopts` ; do
     echo $gen $time
 done )
 
-echo -e "${custom_times[*]} \n${default_times[*]} \n${generated[*]}" | awk '{print $2,$1}'| sort -n | awk '{print $2}'
+echo -e "${custom_times[*]} \n${default_times[*]} \n${generated[*]}" | awk '{print $2,$1}'| sort -nr | awk '{print $2}'
```

Comment:
Don't reverse, or yes reverse?

Reply:
Sorry for confusion--we don't want to change this from master, i had to change this to run the tests in order from least expensive to most expensive so to finish the quicker tests first. The slower tests do timeout or run out of space more so I know i only will be able to run a portion of them. So far I am at 220/330 tests successful and about 10 out of space or out of time. One actually still gets a segmentation fault--but that will be a different PR.